policyengine_us/parameters/gov/states/mt/tax/income/main/capital_gains/rates/reduced_capital_gains/lower.yaml

```diff
@@ -0,0 +1,22 @@
+description: Montana taxes reduced net long-term capital gains by this lower rate, based on filing status.
+metadata:
+  label: Montana lower reduced capital gains tax rate
+  period: year
+  unit: /1
+  breakdown:
+    - filing_status
+  reference:
+    - title: 2023 Montana Form 2 Individual Income Tax Instructions, Net Long-Term Capital Gains Tax Table
+      href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/12/Form_2_2023_Instructions.pdf#page=6
+    - title: 2023 Montana Legislature, Section 1. Section 15-30-2103, MCA, (2)
+      href: https://leg.mt.gov/bills/2023/billhtml/HB0221.htm
```

Comment:
can we check if the bill already included in the new legal code versions?

Reply:
I'll check it.